pkg/daemon/ceph/osd/kms/envs.go

```diff
@@ -30,7 +30,8 @@ import (
 )
 
 var (
-	knownKMSPrefix = []string{"VAULT_", "IBM_"}
+	kmipKMSPrefix  = "KMIP_"
+	knownKMSPrefix = []string{"VAULT_", "IBM_", kmipKMSPrefix}
```

Comment:
its also a good time to make variables for `VAULT` and `IBM`

Reply:
> its also a good time to make variables for `VAULT` and `IBM`

`kmipKMSPrefix` is created as another variable since it is used to add and strip prefix for kmip. 

`VAULT` and `IBM` variables aren't used anywhere else, so I guess we can keep it as it is for now.